textbook/03/5/1/BuiltIn.ipynb

```diff
@@ -6,27 +6,287 @@
    "source": [
```

Comment:
<p>but still important and common -&gt; but are still important and common</p><p> </p><p>logarithmic, and square root functions -&gt; logarithmic, square root functions</p><p> </p><p>Do you think it would be worth mentioning the difference between a <em>library</em> and a <em>module</em> since you used the <code>pandas</code> library earlier? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>